react/util/ui-utils.js

```diff
@@ -83,6 +86,29 @@ export const pickImage = async function (options) {
     return result;
 };
 
+export const isTabletSize = function () {
+    const { height, width } = Dimensions.get("window");
+    if (isLandscape()) {
+        return height > MOBILE_WIDTH && height <= TABLET_WIDTH;
+    }
+    return width > MOBILE_WIDTH && width <= TABLET_WIDTH;
+};
+
+export const isMobileSize = function () {

```

Comment:
I think we should make this condition simply the opposite of `isTabletSize` (`return !isTabletSize()`). I'm afraid that we might end up in a situation where they're both `true` or both `false` if the logic is different. Thoughts?

Reply:
Yes, it makes sense, since it is mutually exclusive